_includes/manuals/3.1/1.2_release_notes.md

```diff
@@ -1,9 +1,34 @@
 ## Release notes for {{page.version}}
 
-This section will be updated prior to the next release.
-
 ### Headline features
 
+#### Improvements and additions to the new Host Details page:
```

Comment:
@amirfefer if you think I should add or remove something then request to comment.